test/e2e/suites/base/suspension_test.go

```diff
@@ -0,0 +1,217 @@
+/*
+Copyright 2024 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package base
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/utils/ptr"
+
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+	"github.com/karmada-io/karmada/pkg/util/names"
+	"github.com/karmada-io/karmada/test/e2e/framework"
+	"github.com/karmada-io/karmada/test/helper"
+)
+
+var _ = ginkgo.Describe("Suspension testing", func() {
+	var deploymentName string
+	var policyName string
+	var deployment *appsv1.Deployment
+	var policy *policyv1alpha1.PropagationPolicy
+
+	ginkgo.BeforeEach(func() {
+		deploymentName = fmt.Sprintf("deployment-%s", rand.String(RandomStrLength))
+		policyName = fmt.Sprintf("policy-%s", rand.String(RandomStrLength))
+
+		deployment = helper.NewDeployment(testNamespace, deploymentName)
+		policy = helper.NewPropagationPolicy(testNamespace, policyName, []policyv1alpha1.ResourceSelector{
+			{
+				APIVersion: deployment.APIVersion,
+				Kind:       deployment.Kind,
+				Name:       deploymentName,
+			},
+		}, policyv1alpha1.Placement{
+			ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+				ClusterNames: framework.ClusterNames(),
+			},
+		})
+		// Set initial suspension state to true
+		policy.Spec.Suspension = &policyv1alpha1.Suspension{
+			Dispatching: ptr.To(true),
+		}
+	})
+
+	ginkgo.AfterEach(func() {
+		framework.RemoveDeployment(kubeClient, deployment.Namespace, deployment.Name)
+		framework.RemovePropagationPolicy(karmadaClient, policy.Namespace, policy.Name)
+	})
+
+	ginkgo.It("Test suspension dispatching with PropagationPolicy", func() {
+		ginkgo.By("Create PropagationPolicy with suspension.dispatching=true", func() {
+			framework.CreatePropagationPolicy(karmadaClient, policy)
+		})
+
+		ginkgo.By("Create Deployment with octopus/user.submitted=false", func() {
+			deployment.Annotations = map[string]string{
+				"octopus/user.submitted": "false",
+			}
```

Comment:
In this case, it seems that this annotation is not being used.

Reply:
just used for updating resourcetemplate

Reply:
Yes, but since the value was not tested in the final evaluation, it becomes meaningless.

Reply:
I understand your scenario. I mean, from the perspective of the current test case, this annotation does not serve the purpose of testing during the process. For example, we did not check whether the deployment resources in the member cluster contain this annotation. Could you please add this logic?